src/TLuaInterpreter.cpp

```diff
@@ -2522,6 +2522,29 @@ int TLuaInterpreter::killTrigger(lua_State* L)
     return 1;
 }
 
+int TLuaInterpreter::remainingTime(lua_State* L)
+{
+	if (!lua_isstring(L, 1)) {
```

Comment:
Please do **NOT** use *TABS* for indentation, set your editor to use **4** *SPACES* instead - if you are using Qt Creator this is done here (`Options` ==> `Text Editor` ==> `Behavior` ==> `Tab policy` which should be `Spaces only` with the `Tab size` and `Indent size` both set to `4`):
![Screenshot_20190405_000819](https://user-images.githubusercontent.com/6163092/55594344-2493e200-5737-11e9-92be-6f853b729be0.png)